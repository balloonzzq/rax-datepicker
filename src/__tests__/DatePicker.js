import {createElement, Component} from 'rax';
import renderer from 'rax-test-renderer';
import DatePicker from '..';

describe('DatePicker', () => {
  it('should render DatePicker', () => {
    const tree = renderer.create(
      <DatePicker
        selectedValue={'2000-01-01'}
        minimumDate={'2000-01-01'}
        maximumDate={'2001-01-01'}
        onDateChange={(date) => {
          console.log('组件date', date);
        }} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});