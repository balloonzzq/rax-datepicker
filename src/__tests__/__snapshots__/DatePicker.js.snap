// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DatePicker should render DatePicker 1`] = `
Object {
  "attributes": Object {
    "defaultValue": "2000-01-01",
    "max": "2001-01-01",
    "min": "2000-01-01",
    "type": "date",
    "value": "2000-01-01",
  },
  "eventListeners": Object {
    "change": [Function],
  },
  "tagName": "INPUT",
}
`;
